packages/ssr-simulator/src/components/gallery/gallery.js

```diff
@@ -112,20 +112,20 @@ export default class Gallery extends React.PureComponent {
     //   import ('./layoutFixer').then(console.warn);
     // }
 
+    const Gallery = (this.isSSR() || !urlParams.useBlueprints) ? ProGallery : LayoutingProGallery
```

Comment:
can you change the ProGallery to blueprintsProGalery? (at the end blueprints/layouting will be the indexes that are left in v3 - in some other naming we choose)

Reply:
Both use blueprint:

layoutingIndex = same api as before, we do the blueprint for you (bigger bundle that includes layouter etc).

blueprintsIndex = you should provide your deconstructed blueprint in the props. (you plan, we build).